### ISSUE TITLE
'New design' in landscape mode overlaps icons

### ISSUE THREAD

**Issue by bazhip** [Maxr1998/XGPM#9]
I have a tablet installed in my dash and I really like the new layout on my phone, but unfortunately when I use it with my tablet, its a big mess and impossible to use. This happens with L and M (just upgraded and double checked) on my Nexus 7 2013 stock image + xposed. Here is a picture:
![4849051019825745573-account_id 1](https://cloud.githubusercontent.com/assets/10350445/22751943/2f11e0c0-edfc-11e6-83ee-daa3091b522f.png)

**Comment by Maxr1998** [comment]
Which Play Music version are you on? I don't even get the new design to apply on my Nexus 6P in landscape, as it uses tablet layouts then.. I will look at the code soon.

**Comment by bazhip** [comment]
I'm on whatever the newest version is. I played around with it a bit and realized that part of the issue is that I have a tiny DPI adjustment in place (I increased it by literally 1) which puts it over the threshold and places my navbar on the right. It's installed in my car's dash, so the screen real estate savings is pretty critical, along with being easier to access. 

When my DPI is stock and the nav bar is on the bottom, the thumbs up and down are in what I assume is the correct position (to the right of the next track button) and don't overlap, but the play/pause button is still ovular and if the 'show equalizer button' option is enabled, it places the equalizer button directly on top of the track listing button.

So, mystery kiiinda solved?
